### ISSUE TITLE
Output normalization of video tokenizer

### ISSUE THREAD

**Issue by witherhoard99** [lucidrains/dreamer4#15]
Hello!

In the video tokenizer, the returned `recon_video` is raw logits.
```py
def decode(latents, height = None, width = None):
    # ...
    recon_video = self.tokens_to_patch(tokens)
    return recon_video

# where self.tokens_to_patch is
self.tokens_to_patch = Sequential(
    Linear(dim, dim_patch),
    Rearrange('b t h w (p1 p2 c) -> b c t (h p1) (w p2)', p1 = patch_size, p2 = patch_size),
)
``` 

I think this would cause issues with MSE and LPIPS loss since they expected normalized inputs, and the input video passed into Video Tokenizer is presumably normalized to [0, 1].

It seems that [other](https://github.com/nicklashansen/dreamer4/blob/main/dreamer4/model.py#L459) implementations use `torch.Sigmoid()`

Unrelated, but it turns out that my previous results that I shared were overfitting significantly.

**Comment by lucidrains** [comment]
@witherhoard99 hey! so it shouldn't be necessary, as the losses will naturally pull the predicted reconstruction into the correct range, but some architectures do constrain it with sigmoid or tanh, like you noticed. I don't think I saw Danijar mention a sigmoid in the paper, but I could add it as hyperparameter for you to try out?
